config/main.py

```diff
@@ -663,6 +663,18 @@ def _get_disabled_services_list(config_db):
 
     return disabled_services_list
 
+def _stop_all_services():
+    # stop all other services
+    for s in _get_sonic_services():
+        (out, err, rv) = clicommon.run_command("sudo systemctl stop {}".format(s), return_err=True)
+        if rv != 0:
+            click.echo("Ignoring an error encountered while stopping service {}.".format(s))
+            log.log_notice("Ignoring an error encountered while stopping service {}. rv={} out={} err={}.".format(s, rv, out.rstrip('\n'), err.rstrip('\n')))
+
+    (out, err, rv) = clicommon.run_command("sudo systemctl stop swss", return_err=True)
```

Comment:
>swss [](http://example.com/codeflow?start=64&length=4)

swss is part of `_get_sonic_services()`, why need stop twice?

Reply:
It is to doubly ensure that swss is down.